tests.sh

```diff
@@ -223,6 +223,13 @@ build_objectivec_osx() {
       --core-only --skip-xcode-ios
 }
 
+build_objectivec_cocoapods_integration() {
+  source ~/.rvm/scripts/rvm
+  rvm use 2.2.3 --install --binary --fuzzy
```

Comment:
Some comment here will likely help in the future.  i.e. - what the source line does, why the custom rvm is installed, etc.